src/components/formControls/textarea/index.tsx

```diff
@@ -12,12 +12,13 @@ export interface Props {
   readOnly?: boolean
   placeholder?: string
   disabled?: boolean
+  fieldError?: boolean
```

Comment:
Per designs, we need to be able to highlight the text area error red in case of a problem

cc: @jenn-rhim